pkgs/by-name/wi/wideriver/package.nix

```
@@ -0,0 +1,45 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  wayland,
+  wayland-scanner,
+  pkg-config,
+}:
+
+stdenv.mkDerivation rec {
```

use the `finalAttrs` pattern
```suggestion
stdenv.mkDerivation (finalAttrs: {
```